docs/schema/index-management/index-lifecycle.md

```diff
@@ -26,28 +26,72 @@ mgmt.commit();
 
 ## Index states and transitions
 
-![States and transitions](index-lifecycle.png)
+The diagram below shows the possible states of an index in JanusGraph.
+Transitions which are not recommended or should be used with caution are shown in grey.
+
+![States and transitions](index-lifecycle.svg){: style="width:100%;"}
 
 ## States (SchemaStatus)
 An index can be in one of the following states:
 
--   **INSTALLED** The index is installed in the system but not yet registered with all instances in the cluster
+**INSTALLED**
 
--   **REGISTERED** The index is registered with all instances in the cluster but not (yet) enabled
+:   The index has been created on the current JanusGraph instance, but it is not yet known to all instances in the cluster.
+    This state is not shown in the diagram above because it is not stable in terms of synchronization between instances.
 
--   **ENABLED** The index is enabled and in use
+**REGISTERED**
 
--   **DISABLED** The index is disabled and no longer in use
+:   The index is known to all instances in the cluster.
+    However, it is not in use yet.
+    In this state, the index contains no data and is not eligible to be used in queries.
```

Comment:
```suggestion
    In this state, the index contains no data and will not be used in queries.
```

(My reasoning here is that JanusGraph is used by many people who aren't native English speakers so we should make the docs easy to understand.)